client-cli/src/main/java/org/eclipse/dataspaceconnector/registration/cli/ParticipantStatusCommand.java

```diff
@@ -0,0 +1,51 @@
+/*
+ *  Copyright (c) 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.registration.cli;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import org.eclipse.dataspaceconnector.registration.client.ApiException;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Model.CommandSpec;
+import picocli.CommandLine.ParentCommand;
+import picocli.CommandLine.Spec;
+
+import java.util.concurrent.Callable;
+
+@Command(name = "status", description = "Get participant status")
+class ParticipantStatusCommand implements Callable<Integer> {
+
+    private static final ObjectMapper MAPPER = new ObjectMapper()
```

Comment:
I see there's mapper in other classes as well. I wonder if it would make sense to extract it somewhere and use the same object mapper everywhere

Reply:
Updated. Thanks